scripts/resume.js

```diff
@@ -0,0 +1,86 @@
+const getResume = filePath => {
+  fetch(filePath)
+    .then(response => {
+      return response.json();
+    })
+    .then(data => {
+      // console.log(data.basics);
```

Comment:
Rather than uncomment console.log statements, remove for final presentation of code. Good that you are doing this though to keep track of your data as you develop!